components/tools/OmeroJava/test/integration/RolesTests.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2017 University of Dundee & Open Microscopy Environment.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package integration;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.annotations.BeforeClass;
+
+import com.google.common.collect.ImmutableMap;
+
+import ome.system.Login;
+import omero.RLong;
+import omero.RString;
+import omero.RType;
+import omero.SecurityViolation;
+import omero.ServerError;
+import omero.model.Dataset;
+import omero.model.IObject;
+import omero.model.Image;
+import omero.model.OriginalFile;
+import omero.model.Permissions;
+import omero.model.PermissionsI;
+import omero.model.Session;
+import omero.sys.EventContext;
+import omero.sys.ParametersI;
+import omero.sys.Principal;
+import omero.util.TempFileManager;
+
+/**
+ * Helper methods class for supporting of "new role" workflow
+ * and security tests.
+ * @author p.walczysko@dundee.ac.uk
+ * @since 5.4.0
+ */
+public class RolesTests extends AbstractServerImportTest {
+
+    private static final TempFileManager TEMPORARY_FILE_MANAGER = new TempFileManager(
+            "test-" + LightAdminRolesTest.class.getSimpleName());
+
+    private File fakeImageFile = null;
```

Comment:
That's the compile error. You refer to this variable in LightAdminPrivilegesTest.java:513 but it's private. Just make it protected by removing the `private` declaration, that should fix the compile error.